src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php

```diff
@@ -209,13 +214,15 @@ public function getNotCalledListeners(Request $request = null): array
         }
 
         $notCalled = [];
-        foreach ($allListeners as $eventName => $listeners) {
-            foreach ($listeners as $listener) {
-                if (!\in_array($listener, $calledListeners, true)) {
-                    if (!$listener instanceof WrappedListener) {
+
+        foreach ($allListeners as $eventName => $eventListeners) {
+            foreach ($eventListeners as $priority => $priorityListeners) {
+                foreach ($priorityListeners as $listener) {
```

Comment:
This structure won't fit the "else" block above.

Reply:
Good catch